src/types/index.ts

```diff
@@ -82,7 +82,7 @@ export interface ReserveToken extends Token {
   listed?: boolean
 }
 
-export interface BigNumberMap {
+export interface bigintMap {
```

Comment:
for renaming use BigIntMap or leave it as is, there are multiple files that use this type, build fails with current change

Reply:
Not too sure why I changed this... I've reverted it